recipes/hypnotoad/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hypnotoad" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 
```

Comment:
You're going to need the hash. You can get it off of the pypi website:

https://pypi.org/project/hypnotoad/#files